Doc/library/socket.rst

```diff
@@ -352,6 +352,12 @@ Constants
    defined then this protocol is unsupported.  More constants may be available
    depending on the system.
 
+.. data:: AF_UNSPEC
+
+   The value :const:`AF_UNSPEC` indicates that
+   :func:`getaddrinfo` should return socket addresses for any
+   address family (either IPv4 or IPv6, for example) that can
+   be used with node and service.
```

Comment:
Is it okay to copy and paste from another source?

Reply:
I think so, it is GNU licensed:

```
» man getaddrinfo | grep PF_UNSPEC
                    is set to PF_UNSPEC, it means the caller will accept any
     provided a struct addrinfo with ai_family set to PF_UNSPEC and all other
     example, “192.0.2.1” ) and ai_family is set to PF_UNSPEC or PF_INET6,
```

If you have concerns, I can give it a bit of a remix :)